templates/cube/index.html

```diff
@@ -9,34 +9,154 @@
+            loading="auto"
+            ) | safe
+          }}
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Ubuntu Desktop Essentials Microcertification</h3>
+          <div class="p-matrix__desc">
+            <p>Demonstrate your knowledge of Ubuntu Desktop administrative essentials. Focus on package management, system installation, data gathering, and managing printing and displays.</p>
+            <a
+              href="/cube/contact-us"
+              class="p-button--positive cube-access js-invoke-modal"
+              data-testid="interactive-form-link"
+              >Sign up</a
+            >
+          </div>
+        </div>
+      </li>
+      <li class="p-matrix__item">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/64422ff3-Virtualisation.svg",
+          alt="Ubuntu Server Essentials badge",
+          width="51",
+          height="60",
+          hi_def=True,
+          attrs={"class":"p-matrix__img"},
+          loading="auto"
+          ) | safe
+          }}
+        <div class="p-matrix__content">
+          <h3 class="p-matrix__title">Ubuntu Server Essentials Microcertification</h3>
+          <div class="p-matrix__desc">
+            <p>Illustrate your knowledge of common Ubuntu Server administrative tasks and troubleshooting. Topics include job control, performance tuning, services management, and Bash scripting.</p>
+            <a
+              href="/cube/contact-us"
+              class="p-button--positive cube-access js-invoke-modal"
+              data-testid="interactive-form-link"
+              >Sign up</a
+            >
+          </div>
+        </div>
+      </li>
     </ul>
-    <a
-      href="/cube/contact-us"
-      class="p-button--positive cube-access js-invoke-modal"
-      data-testid="interactive-form-link"
-      >Sign up</a
-    >
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <p class="p-heading--4">Exam features:</p>
+      <ul class="p-list">
+        <li class="p-list__item">Multiple choice questions</li>
+        <li class="p-list__item">3-5 practical hands-on lab questions</li>
+      </ul>
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <p class="p-heading--4">Or take a single exam covering all three microcertifications:</p>
+      <p class="p-heading--4">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/7bf909cc-Admin.svg",
+          alt="Canonical Ubuntu Essentials badge",
+          width="51",
+          height="60",
+          hi_def=True,
+          loading="auto",
+          attrs={"style": "vertical-align:middle"}
+          ) | safe
+        }}
+        Canonical Ubuntu Essentials single exam
+      </p>
+      <a
+        href="/cube/contact-us"
+        class="p-button--positive cube-access js-invoke-modal"
+        data-testid="interactive-form-link"
+        >Sign up</a
+      >
+    </div>
   </div>
```

Comment:
I wasn't sure how this section should be layed out.

Reply:
@mrgnr  I would put this new section into its own p-strip so it's on a white background, separate from the grey strip above it